src/resources/i18n/translations/en.json

```
@@ -260,20 +260,20 @@
   "Custom API domain": "Custom API domain",
   "Custom Feeds": "Custom Feeds",
   "Custom RSS feeds to parse in parallel": "Custom RSS feeds to parse in parallel",
-  "Custom RSS feeds to parse in parallel helper text": "Parsing too many feeds in parallel may cause issues on some devices. 3 is recommended.",
+  "Custom RSS feeds to parse in parallel helper text": "Parsing more than three feeds at once may cause issues on some devices.",
   "Custom web domain": "Web domain",
-  "Custom web domain subtext": "Custom domains to use an alternate server for your app data.",
+  "Custom web domain subtext": "These allow choice of app data server.",
   "Dark Mode": "Dark",
   "Data Settings": "Data Settings",
-  "Default downloaded episode limit for each podcast": "Max downloaded episodes for each podcast",
+  "Default downloaded episode limit for each podcast": "Limit",
   "Delete": "Delete",
   "DELETE": "Delete",
   "Delete Account": "Delete account",
   "Delete All Downloaded Episodes": "Delete all downloaded episodes",
```

```suggestion
  "Delete All Downloaded Episodes": "Delete all downloads",
```
Assuming it is also clips. (In Settings → Downloads)